src/DbSQLiteAdapter/CMakeLists.txt

```diff
@@ -2,9 +2,10 @@ cmake_minimum_required(VERSION 3.2)
 
 # Find external dependencides
 find_package(DbAdapterInterface)
+find_package(sqleet)
 
 # Configure preprocessor definitions
-add_compile_options(-DBOOST_ALL_DYN_LINK -DSQLITE_HAS_CODEC)
```

Comment:
test\DbSQLiteAdapterTest\CMakeLists.txt
and
test_package\CMakeLists.txt

still have -DBOOST_ALL_DYN_LINK which could be deleted

Reply:
You are right, corrected